components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -574,14 +561,11 @@ export default class AdvancedSettingsDisplay extends React.Component {
 }
 
 AdvancedSettingsDisplay.propTypes = {
-    user: PropTypes.object,
     updateSection: PropTypes.func,
-    updateTab: PropTypes.func,
```

Comment:
Not related to this PR but just remove this unused props. Will submit another PR to do the same with other user settings.